Brokkr/src/main/include/commands/ArmPosition.h

```diff
@@ -30,5 +32,6 @@ class ArmPosition
 
   private:
   Arm& mArm;
+  frc2::PIDController mArmControl{0, 0, 0};
```

Comment:
Initialize mArmController in the cpp file instead of the header.  The constants for PID can be defined in the cpp file as well.